.meta/dietpi-imager

```diff
@@ -115,19 +115,22 @@
 		OUTPUT_IMG_NAME=$G_WHIP_RETURNED_VALUE
 
 		# Check for existing file, in case offer backup. Skip if source is image file and matches output file already.
-		if [[ /root/$OUTPUT_IMG_NAME.img != $FP_SOURCE_IMG && -f /root/$OUTPUT_IMG_NAME.img ]]; then
+		if [[ $OUTPUT_IMG_NAME.img != $FP_SOURCE_IMG && -f $OUTPUT_IMG_NAME.img ]]; then
```

Comment:
`$FP_SOURCE_IMG` is full file path, hence must be checked against `/root/$OUTPUT_IMG_NAME.img`.

Reply:
Yes, you're right

Reply:
Change Undone